src/UpdatesPublisher.js

```diff
@@ -0,0 +1,76 @@
+const {createProducer} = require('windbreaker-service-util/queue')
+const ChangesStream = require('changes-stream')
+const Promise = require('bluebird')
+
+class UpdatesPublisher {
+  constructor (options) {
+    this.producerOptions = options.producerOptions
+    this.amqUrl = options.amqUrl
+    this.registryUrl = options.registryUrl
+    this.logger = options.logger
```

Comment:
For the fields that you append to your publisher, start them with an underscore to signify that they are supposed to be private and not really exposed to clients that are using the publisher.

```
constructor (options) {
  this._producerOptions = options.producerOptions
  this._amqUrl = options.amqUrl
  // etc..
}
```